plasma_framework/docs/integration-docs/integration-doc.md

```diff
@@ -0,0 +1,283 @@
+# Introduction
+This document attempts to collate all of the information necessary to interact with the Plasma ALD framework. For more detailed information on the concepts involved, see the following documents:
+
+- [High level design of the Plasma Abstract Layer](https://docs.google.com/document/d/1PSxLnMskjqje4MksmW2msSSg-GtZoBSMNYey9nEDvt8)
+- [Tesuji Plasma Blockchain Design](https://github.com/omisego/elixir-omg/blob/master/docs/tesuji_blockchain_design.md)
+- [Solidity contract documentation](https://github.com/omisego/plasma-contracts/blob/master/plasma_framework/docs/PlasmaFramework.md)
+
+# Block submission
+Only the operator can submit blocks. The data submitted is the root of the merkle tree containing all the transactions in the block. To submit a block, call
+```
+PlasmaFramework.submitBlock(blockRoot);
+```
+On success emits a `BlockSubmitted` event
+```
+event BlockSubmitted(
+    uint256 blockNumber
+);  
+``` 
+[See contract docs](../BlockController.md#submitblock)
+
+# Transactions
+Transactions are composed of inputs and outputs. An input is simply a pointer to the output of another transaction. 
+
+Transactions that have been included in a block have a position which is the number of the block it's in and its index in that block. For example the fourth transaction in block 5000 has a position of `(5000, 3)`.
+
+The position of the outputs of a transaction can be obtained by including the index of the output in the transaction. So the position of the second output of the transaction in the above example would be `(5000, 3, 1)`.
+
+## Transaction type and output type
+
+The Abstract Layer Design introduces the concept of Transaction Type and Transcation Output Type. Each Transaction Type and Transcation Output Type can define different rules about how to spend funds.
+
+## Transaction format
+Even though there are different types of transaction and transaction outputs, all transactions have the same basic format:
```

Comment:
this might mislead that ALD actually requires all transactions have the same data structure which is something not really true and I would not want to limit.

I might consider to use the `Wire Transaction format` to make it easier to link with current `WireTransaction` library. Also, make it clear that this format is expected to support Payment + DEX related tx types (which is of our highest priority) but sadly not supporting ERC721.